### PR TITLE
[DOCS-160] Co-branded Pentest Reports

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
@yogipetkar Please review the updates describing co-branded pentest reports.

### CODE REVIEW

**Comment by mjang-cobalt** [comment]
@yogipetkar You can review how this will look in our docs at:

- https://deploy-preview-137--cobalt-docs.netlify.app/getting-started/checklist/#co-branded-reports
- https://deploy-preview-137--cobalt-docs.netlify.app/getting-started/glossary/#organization-owner (Select "Learn more" to see the new content)

**Review by mjang-cobalt — APPROVED** [review]
Thanks, @ana-dashuk-cobalt. Just a reminder: don't merge this PR until the engineering changes are merged on June 30.

**Comment by yogipetkar** [comment]
Hi @ana-dashuk-cobalt - Please take a look at following ZD article I have written. To enable partner attribute they need to reach out to Cobalt. Once such attribute is enabled they can managed to use the feature.
https://docs.google.com/document/d/1jS8FqJNYnpaoLXcc52XvRQrtnOT-mr7ITiip-D6spUA/edit?usp=sharing

File: content/en/Getting started/checklist.md
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/checklist.md:82]: @yogipetkar I've added more details about enabling co-branded reports. Please review.

File: content/en/Getting started/checklist.md
Comment (yogipetkar) [inline content/en/Getting started/checklist.md:69]: Once we've processed your request, you have partner attributes, which ....

File: content/en/Getting started/checklist.md
Comment (yogipetkar) [reply content/en/Getting started/checklist.md:82]: One minor comment added in line.

File: content/en/Getting started/checklist.md
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/checklist.md:69]: @yogipetkar Updated. Please approve the PR if we are ready to go with this version.

**Pull request merged**
